src/test/java/LoginTests.java

```diff
@@ -1,25 +1,21 @@
-import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.chrome.ChromeDriver;
-import org.openqa.selenium.chrome.ChromeOptions;
-import org.testng.Assert;
 import org.testng.annotations.Test;
+import pages.LoginPage;
+import org.testng.Assert;
 
-import java.time.Duration;
-
-public class LoginTests extends BaseTest {
+public class LoginTests extends BaseTest{
     @Test
-    public void LoginEmptyEmailPasswordTest() {
-
-//      Added ChromeOptions argument below to fix websocket error
-        ChromeOptions options = new ChromeOptions();
-        options.addArguments("--remote-allow-origins=*");
-
-        WebDriver driver = new ChromeDriver(options);
-        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
-
-        String url = "https://testpro.io/";
-        driver.get(url);
-        Assert.assertEquals(driver.getCurrentUrl(), url);
-        driver.quit();
+    public void validLoginTest1 () {
+        LoginPage loginPage = new LoginPage(getDriver());
+        openLoginUrl();
+        loginPage.provideEmail("beomseo.park@testpro.io").providePassword("te$t$tudent").clickSubmit();
+        Assert.assertEquals(getDriver().getCurrentUrl(), url);
+    }
+    @Test
+    public void validLoginTest2 () {
+        LoginPage loginPage = new LoginPage(getDriver());
+        openLoginUrl();
+        loginPage.loginUrl();
+        loginPage.login();
```

Comment:
Please, remove code redundancies

Don't repeat yourself! ( In Java, it means don't write the same code repeatedly. )


![image](https://github.com/Alexer0105/A47-QA-Automation-Framework/assets/104761482/f817f88b-4cc6-411d-add4-527658f627e7)